src/client/proxy/socks.cc

```diff
@@ -49,606 +49,725 @@
+  void SOCKSRequestSuccess();
 
-  void SentSocksFailed(
+  void SentSOCKSFailed(
       const boost::system::error_code & ecode);
 
-  void SentSocksDone(
+  void SentSOCKSDone(
       const boost::system::error_code & ecode);
 
-  void SentSocksResponse(
+  void SentSOCKSResponse(
       const boost::system::error_code & ecode);
 
   void HandleStreamRequestComplete(
       std::shared_ptr<kovri::client::Stream> stream);
 
-  std::uint8_t m_sock_buff[SOCKS_BUFFER_SIZE];
-  std::shared_ptr<boost::asio::ip::tcp::socket> m_sock;
-  std::shared_ptr<kovri::client::Stream> m_stream;
-  std::uint8_t *m_remaining_data;  // Data left to be sent
-  std::uint8_t m_response[7+MAX_SOCKS_HOSTNAME_SIZE];
-  address m_address;  // Address
-  std::size_t m_remaining_data_len;  // Size of the data left to be sent
-  std::uint32_t m_4aip;  // Used in 4a requests
-  std::uint16_t m_port;
-  std::uint8_t m_command;
-  std::uint8_t m_parseleft;  // Octets left to parse
-  authMethods m_authchosen;  // Authentication chosen
-  addrTypes m_addrtype;  // Address type chosen
-  socksVersions m_socksv;  // Socks version
-  cmdTypes m_cmd;  // Command requested
-  state m_state;
+  std::uint8_t m_SocketBuffer[SOCKSBufferSize];
+  std::shared_ptr<boost::asio::ip::tcp::socket> m_Socket;
+  std::shared_ptr<kovri::client::Stream> m_Stream;
+  std::uint8_t *m_RemainingData;  // Data left to be sent
+  std::uint8_t m_Response[7+MaxSOCKSHostSize];
+  Address m_Address;  // Address
+  std::size_t m_RemainingDataLen;  // Size of the data left to be sent
+  std::uint32_t m_SOCKS4aIP;  // Used in 4a requests
+  std::uint16_t m_Port;
+  std::uint8_t m_CommandNum;
+  std::uint8_t m_ParseLeft;  // Octets left to parse
+  AuthMethods m_AuthChosen;  // Authentication chosen
+  AddressTypes m_AddressType;  // Address type chosen
+  SOCKSVersions m_SOCKSVersion;  // Socks version
+  CommandTypes m_Command;  // Command requested
+  State m_State;
 
  public:
   SOCKSHandler(
       SOCKSServer* parent,
-      std::shared_ptr<boost::asio::ip::tcp::socket> sock)
+      std::shared_ptr<boost::asio::ip::tcp::socket> socket)
       : I2PServiceHandler(parent),
-        m_sock(sock),
-        m_stream(nullptr),
-        m_remaining_data(nullptr),
-        m_remaining_data_len(0),
-        m_port(0),
-        m_command(0),
-        m_authchosen(AUTH_UNACCEPTABLE),
-        m_addrtype(ADDR_IPV4),
-        m_socksv(SOCKS5),
-        m_cmd(CMD_CONNECT) {
-          m_address.ip = 0;
-          EnterState(GET_SOCKSV);
+        m_Socket(socket),
+        m_Stream(nullptr),
+        m_RemainingData(nullptr),
+        m_RemainingDataLen(0),
+        m_Port(0),
+        m_CommandNum(0),
+        m_AuthChosen(Invalid),
+        m_AddressType(IPv4),
+        m_SOCKSVersion(SOCKS5),
+        m_Command(Connect) {
+          m_Address.ip = 0;
+          EnterState(GetSOCKSVersion);
         }
   ~SOCKSHandler() { Terminate(); }
-  void Handle() { AsyncSockRead(); }
+  void Handle() { AsyncSocketRead(); }
```

Comment:
Anytime you see inline formatting like this, and you're doing style refactoring, feel free to newline.